glom/test/test_error.py

```diff
@@ -0,0 +1,18 @@
+
+import pytest
+
+from glom import glom, T, GlomError
+
+@pytest.mark.xfail
```

Comment:
I think these might actually be more of a place for `with pytest.raises` than xfails.